spec/lib/utils/job-utils/command-parser-spec.js

```diff
@@ -302,6 +302,35 @@ describe("Task Parser", function () {
             });
         });
 
+        it("should parse lshw -json when there are multiple nics ", function () {
+            // Test cases where pci network string resembles "network:0", "network:1", etc.
+            // rather than just "network".
+            var lshwCmd = 'sudo lshw -json';
+            var tasks = [
+                {
+                    cmd: lshwCmd,
+                    stdout: stdoutMocks.lshwOutputMultiNic,
+                    stderr: '',
+                    error: null
+                }
+            ];
+
+            return taskParser.parseTasks(tasks)
+            .spread(function (result) {
+                expect(result.error).to.be.undefined;
+                expect(result.store).to.be.true;
+                expect(result.lookups).to.deep.equal(
+                    [
+                        { mac: 'f8:bc:12:0b:0b:40' },
+                        { mac: 'f8:bc:12:0b:0b:41' }
+                    ]
+                );
+                expect(_.isEqual(result.data,
+                    JSON.parse(stdoutMocks.lshwOutputMultiNic))).to.be.true;
```

Comment:
Yep, I'll fix it. I copy/pasted the above test, which was written before we knew the Chai framework very well, haha :)